django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from ulid2 import generate_ulid_as_uuid
+
+
+class Comment(models.Model):
+    commented_object = GenericForeignKey()
+    content_type = models.ForeignKey(
```

Comment:
Continuing naming nitpicks, rather than `content_type`, this should probably be named `thread_content_type` and `object_id` to `thread_object_id` respectively. Otherwise it might be a bit confusing what the fields stand for if you don't know how generic foreign keys work or aren't aware these fields are related to the generic foreign key.

Reply:
the `GenericForeignKey()` field takes attributes for specifying the name of the content type and object ID fields

Reply:
Because I ended up renaming the fields to the standards I also removed the arguments as it defaults to the current field names